chapter_optimization/adadelta_vn.md

```diff
@@ -20,7 +20,7 @@ Adadelta là một biến thể khác của AdaGrad.
 Điểm khác biệt chính là Adadelta giảm mức độ thay đổi của tốc độ học theo toạ độ.
```

Comment:
```suggestion
Điểm khác biệt chính là Adadelta giảm mức độ mà tốc độ học sẽ thay đổi với các toạ độ.
```
Viết như bây giờ mình đọc hiểu nhầm thành `it decreases the amount _of change in_ learning rate  _per_ coordinates.`